### ISSUE TITLE
3.5.2 - helm chart config is missing pgbench-job.json and is inconsistent with release tarball

### ISSUE THREAD

**Issue by PowerStateFailure** [CrunchyData/postgres-operator#670]
**Describe the bug**
Helm chrat config dir is inconsistent with release tarball:

1. Missed `pgbench-job.json`, which causes operator to fail during startup:
```
time="2019-03-30T04:40:44Z" level=debug msg="loading path [/pgo-config/pgbench-job.json]"
time="2019-03-30T04:40:44Z" level=error msg="open /pgo-config/pgbench-job.json: no such file or directory"
time="2019-03-30T04:40:44Z" level=error msg="error loading template path=/pgo-config/pgbench-job.jsonopen /pgo-config/pgbench-job.json: no such file or directory"
panic: open /pgo-config/pgbench-job.json: no such file or directory

goroutine 1 [running]:
github.com/crunchydata/postgres-operator/util.LoadTemplate(0x119232b, 0x1c, 0xc000210880)
        /opt/cdev/src/github.com/crunchydata/postgres-operator/util/util.go:96 +0x46c
github.com/crunchydata/postgres-operator/operator.Initialize()
        /opt/cdev/src/github.com/crunchydata/postgres-operator/operator/common.go:159 +0x70d
main.main()
        /opt/cdev/src/github.com/crunchydata/postgres-operator/postgres-operator.go:65 +0x1bf
```
2. Config files differ from ones in release tarball (`postgres-operator.3.5.2.tar.gz`), see diff: https://gist.github.com/PowerStateFailure/1079fa92ce365206ebd05d175e55013b
    (here `a` stands for helm chart configs, `b` - for release tarball configs)

Which config files should be used?

**To Reproduce**
- Install Helm chart from tarball `postgres-operator-helm-chart.3.5.2.tar.gz`

**Expected behavior**

**Screenshots**

**Please tell us about your environment:**

* Operating System: Debian 9
* Where is this running ( Local, Cloud Provider): Local
* Storage being used (NFS, Hostpath, Gluster, etc): Local
* Container Image Tag: 3.5.2
* PostgreSQL Version: 10
* Platform (Docker, Kubernetes, OpenShift): Kubernetes
* Platform Version: 1.14

**Additional context**

**Comment by Deepak-Routray** [comment]
@PowerStateFailure Any luck with this? 
I have been facing the same issue with version co_image-tag "centos7-3.5.2" for some time now. 
I worked around this by reverting the image tag to "centos7-3.5.1" in pgo.yaml and values.yaml. 
FYI I am using Helm Chart for deploying. I tried bumping up the version to latest one and tweaking around some configs. But unfortunately that did not work.
Hope this helps.

Deepak

**Comment by devlounge** [comment]
This operator and its repo are really messy IMO and do not follow how other operators are written.
Also, so many files have a mix of tabs and spaces, dirty.
I keep trying to make it work, but honestly I feel like giving up...

**Comment by jmccormick2001** [comment]
the helm chart is going away starting in 4.0 release which is planned for 
next week, instead we are investing in an ansible approach to an installation
method apart from the developer/default install.  If someone wanted to write a new
helm chart for 4.0 we would welcome it, it would require a complete rewrite most
likely to deal with all the 4.0 changes that have been made.

**Comment by PowerStateFailure** [comment]
> @PowerStateFailure Any luck with this?

@Deepak-Routray Sorry for the late reply (too long vacation).
Yes, I've repaired Helm chart by copying `pgbench-job.json` from release archive, and that's all - operator works fine.

@jmccormick2001 Would you accept any PRs on Helm chart for 3.5.*? I'm planning to update chart to support 4.0 branch of operator so at first I've synced chart with version 3.5.4

**Comment by jmccormick2001** [comment]
yes, a PR for that would be very helpful.  thanks.

**Comment by jpigottj** [comment]
I'd also be interested in a helm chart for 4.0.x. I tried to take 3.5.4 chart and apply to version 4.0.0 but in addition to issue with `pgbench-job.json`, I did also encounter  some issues with CRD group names and needing `PGO_OPERATOR_NAMESPACE` env var to be specified for the operator containers. With these changes creation of a pgcluster works but sure there is other functionality I didn't test that may have issues so would be great to have supplied helm charts for version 4.0.x.
